Packs/VMware/Integrations/VMware/VMware.yml

```diff
@@ -17,6 +17,11 @@ configuration:
   additionalinfo: Username and password used to login into the system.
   displaypassword: ""
   hiddenusername: false
+- name: redirect_std_out
```

Comment:
lets give a default of false in the yml and also the code

Reply:
It's a checkbox it's False by default.
Anyway there's also a fault false in the code.